### DISCUSSION TITLE
Segmentation and mesh with Fieltrip

### DISCUSSION THREAD

**Discussion by saracinelli** [simnibs/simnibs#381] [7.Creating head models]
Hello, I would like to ask how to create the file required of the mesh from the segmentation and the mesh obtained with fieldtrip software (with the logical masks).
 Thank you,
Sara

**Comment by jduemose** [comment]
Hi Sara, I am not sure I follow: do you want to create a segmentation mask (yourself or using fieldtrip) and then create a mesh using simnibs? Or do you want to use a mesh created with simnibs in fieldtrip?

**Reply by saracinelli** [reply]
Thank you for the answer and sorry for being not clear. I would like to use the segmentation and the mesh which I obtained from Fieltrip and using this mesh in simnibs. Is it possibile? The type of method used in mesh has to be a particular one (ex. triangles, tetrahedral,ecc.)?
Thank you

**Reply by jduemose** [reply]
Simnibs uses tetrahedral meshes so if this is what you have from fieldtrip you should in principle be able to use it as long as you save it in the correct format (.msh file). You can get a feel for it [here](https://github.com/simnibs/simnibs/blob/master/simnibs/matlab_tools/mesh_load_gmsh4.m) and then save it using `mesh_save_gmsh4`. I know that fieldtrip reads `msh` files --- not sure if it also saves them. The indices of the different compartments goes in the *regions field.

Although saving the mesh will in principle work, you may find that it is tricky to get it to work. But you can give it a shot. Otherwise, you can save the segmentation from fieldtrip and mesh it using the `meshmesh` function in simnibs. See [this](https://simnibs.github.io/simnibs/build/html/tutorial/advanced/custom_meshing.html) tutorial for an example of how to do this and how to run a simulation. You can see the standard conductivities used for different labeled compartments [here](https://simnibs.github.io/simnibs/build/html/documentation/conductivity.html) (that is, save your segmentation with these indices to use these conductivities, e.g., white matter=1) and otherwise supply your own for other labels.
